pkg/beholder/chip_ingress_emitter.go

```
@@ -0,0 +1,97 @@
+package beholder
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/chipingress"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+type ChipIngressEmitter struct {
+	client chipingress.ChipIngressClient
+}
+
+func NewChipIngressEmitter(client chipingress.ChipIngressClient) Emitter {
+
+	if client == nil {
+		panic("chip ingress emitter client cannot be nil")
```

We construct this client in loops and in the core node on startup, so we can't panic. Can we return an error instead?

---

The way the code is currently written should not hit this code path but definitely it should not panic 
https://github.com/smartcontractkit/chainlink-common/pull/1106/files#diff-74ce7f3d5474f67956a5d9ba5779b478b715235a0d33e8826eb99b146664ae4dR210-R217